Added docs about branches. (#102)
avohq/docs: pages/workspace/branches.mdx

---
layout: 'guide'
title: 'Branches'

> # abstract: 'Learn about branched workflows'

Reviewer: How about something like this?
```suggestion
abstract: 'Seamlessly work together on multiple fronts using branches'
```

---

> import Link from '../../components/Link';
> 
> 

Reviewer: I think that we can skip this
```suggestion
```

## What are branches?

> Avo offers branched workflows similar to git, to enable seamless collaboration between and within teams in your company. That means when you are making changes to your tracking plan, you can do it on a branch such that you can play around with your changes without affecting the source of truth on the main branch.

Reviewer: Not to get all into Minto, but I'd love to start _Why_ here. Why would someone need branches? Perhaps something like this: 
```suggestion
In Avo you can collaborate in realtime on defining, expanding and implementing your tracking plan. But you might want to try out some changes or make sure that a change is implemented before you change your tracking plan source of truth. To capture this work in progress you can use branches.
```
